.github/workflows/build.yml

```diff
@@ -80,6 +84,36 @@ jobs:
         with:
           flags: dotnet
 
+  build-windows-setup:
```

Comment:
This should probably be added to the release as well (lower down in this file)

Reply:
Yeah intentionally left it out until we validate the installer more.